pkg/cmd/eventlistener/describe.go

```diff
@@ -277,3 +293,22 @@ func isBindingSpecExist(bindings []*v1alpha1.EventListenerBinding) bool {
 	}
 	return specExist
 }
+
+func isBindingNameExist(bindings []*v1alpha1.EventListenerBinding) bool {
+	nameExist := false
+	for _, j := range bindings {
+		if j.Name != "" {
+			nameExist = true
+		}
```

Comment:
Would it make sense to just return here?

```suggestion
		if j.Name != "" {
			return true
		}
```